src/main/java/school/hei/haapi/endpoint/OwnCloudConf.java

```diff
@@ -0,0 +1,49 @@
+package school.hei.haapi.endpoint;
```

Comment:
I don't think this is a good idea to put in endpoint package, it is more of a service.owncloud file than an endpoint one